tests/components/sonarr/test_calendar.py

```diff
@@ -0,0 +1,50 @@
+"""Test the Sonarr calendar entity."""
+from http import HTTPStatus
+
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_calendar(
```

Comment:
Please also add a test of the sonarr calendar entity state in the state machine.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations